run_brer/tests/test_run_config.py

```diff
@@ -69,6 +69,8 @@ def test_run_config(tmpdir, data_dir):
         # Include a test for kwarg handling.
         rc.run(threads=num_cpus)
 
+        # Include a test to override alpha=0 fail
+        rc.run_data.set(alpha=1.0)
```

Comment:
Looks like we need to provide the pair name, as well. I think that derives from the pair_data.json file.

```
        for key, value in kwargs.items():
            # If a restraint name is not specified, it is assumed that the parameter is
            # a "general" parameter.
            if not name:
                if key in self.general_params.get_requirements():
                    self.general_params.set(key, value)
                else:
                    raise ValueError(
                        'You have provided a name; this means you are probably trying '
                        'to set a '
>                       'pair-specific parameter. {} is not pair-specific'.format(key))
E                   ValueError: You have provided a name; this means you are probably trying to set a pair-specific parameter. alpha is not pair-specific
```

Also, we'll need to do this in a couple of more places, I expect. (Wherever we use a value that was trained with an astronomical `tolerance`.) It's fine to just play "whack-a-mole" and just respond to the tests that are failing.

Reply:
In test_run_config.py, we set up the dictionary with: 
`        rc = RunConfig(**config_params)
        rc.run_data.set(A=5,
                        tau=0.1,
                        tolerance=10000,
                        num_samples=2,
                        sample_period=0.001,
                        production_time=10000.)`
                        and then `rc.run()`.  Are you saying that we need to assign a specific residue-pair site to the alpha value?

Reply:
It looks like that error message is worded weirdly, and we should update it. But the upshot is that 'alpha' is evidently not a "general" parameter, and so the pair to which it is to apply needs to be named. https://kassonlab.github.io/run_brer/_build/html/user_api.html#run_brer.run_data.RunData.set

Reply:
At this line, we need to use the version of `set` that includes the pair-name to which the parameter applies.

Reply:
see https://github.com/kassonlab/run_brer/pull/23/files#r855359959